pkg/health/health.go

```diff
@@ -111,17 +111,6 @@ func GetHealthCheckFunc(gvk schema.GroupVersionKind) func(obj *unstructured.Unst
 		case kube.DaemonSetKind:
 			return getDaemonSetHealth
 		}
-	case "extensions":
-		switch gvk.Kind {
-		case kube.DeploymentKind:
-			return getDeploymentHealth
-		case kube.IngressKind:
```

Comment:
TODO: Need to add this back since extensions Ingress is still available/served in K8s 1.21 
> The v1.22 release will stop serving the following deprecated API versions in favor of newer and more stable API versions:

> Ingress in the extensions/v1beta1 API version will no longer be served
        Migrate to use the networking.k8s.io/v1beta1 API version, available since v1.14. Existing persisted data can be retrieved/updated via the new version.